.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
++**I'm submitting a ..**
++- [ ] Bug Report
++- [ ] Feature Request
++
++**Current behavior:**
++<!-- Describe the bug and how it creeped in. -->
++
++**Expected behavior:**
++<!-- Describe what the behavior would be without the bug. -->
++
++**Steps to reproduce:**
++<!-- If you are able to illustrate the bug or feature request with an example, please provide steps to reproduce -->
++
++**Related code:**
```

Comment:
Not sure if we need this section. Bugs will not necessarily come from developers.